release_tester/arangodb/starter/manager.py

```diff
@@ -80,7 +81,7 @@ def __init__(
         # self.moreopts += ["--starter.disable-ipv6=false"]
         # self.moreopts += ["--starter.host=127.0.0.1"]
 
-        #if self.cfg.enterprise:
```

Comment:
Can we remove this debug code which is no more required?

Reply:
I'd like to keep it since it caries useful combinations we may need again for debugging. 
It sometimes takes quiet a bit of searching + a good idea what to search to build these.